clients/lsp-clangd.el

```diff
@@ -237,7 +237,10 @@ returned to avoid that the echo area grows uncomfortably."
 
 (defun lsp-clangd-to-other ()
```

Comment:
To be consistent with `projectile-find-other-file`, this should be named `lsp-clangd-find-other-file`.

Reply:
Makes sense. Is consistency with projectile something that I should generally keep in mind? What about project.el that seems to be gaining feature parity?

Reply:
That is not something you need to keep in mind especially; we don't depend on `projectile`. This is just about having naming that is consistent with the rest of Emacs in general.

Reply:
Generally, for the extension methods, we follow the name of the server method. In this case, something like lsp-clangd-switch-source-header should do.

Reply:
@yyoncho but in this case I'd argue that there is benefit to be had from being consistent with the `projectile` naming scheme:
1. That name is good and intuitive (IMHO)
2. More familiarity: users of `projectile` could discover this function more easily trough `M-x`